### PR TITLE
Bootstrap package publishing

### PR DESCRIPTION
**Pull request by mheap**
Add auto-publish support using `auto`

<details>
  <summary>📦 Published PR as canary version: <code>2.1.1--canary.30.cd106ef.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @kong/sdk-portal-js@2.1.1--canary.30.cd106ef.0
  # or 
  yarn add @kong/sdk-portal-js@2.1.1--canary.30.cd106ef.0
  ```
</details>

### CODE REVIEW

**Review by andrewwylde — APPROVED** [review]
LGTM, I presume the first release will go off of the version `2.1.0`?

**Comment by mheap** [comment]
Yep! That's the expectation

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+name: Release
+
+on: [push]
```
Comment (andrewwylde) [inline .github/workflows/release.yaml:3]: Don't we need the branch filter specified as well? I figured we would want it to be only on pushes to `main`

Comment (mheap) [reply .github/workflows/release.yaml:3]: Yep that’s an oversight

Comment (mheap) [reply .github/workflows/release.yaml:3]: It also releases canary builds on push to PR

**Pull request merged**